packages/react-components/src/components/VideoEffects/VideoBackgroundEffectsPicker.tsx

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IStyle, Label, mergeStyles, Stack } from '@fluentui/react';
+import React from 'react';
+import { chunk } from '../utils';
+import { _VideoEffectsItem, _VideoEffectsItemProps } from './VideoEffectsItem';
+
+/**
+ * Props for {@link _VideoBackgroundEffectsPicker}
+ * @internal
+ */
+export interface _VideoBackgroundEffectsPickerProps {
+  /**
+   * The options to display in the picker.
+   */
+  options: _VideoBackgroundEffectChoiceOption[];
+
+  /**
+   * The key of the current selected Video Background Effect.
+   * If you provide this, you must maintain selection state by observing onChange events and passing a new value in when changed.
+   */
+  selectedEffectKey?: string;
+
+  /**
+   * Callback to invoke when a Video Background Effect is selected.
+   * @param effectKey - The key of the Video Background Effect that was selected.
+   */
+  onChange?: (effectKey: string) => void;
+
+  /**
+   * The label to display for the picker.
+   */
+  label?: string;
+
+  /**
+   * The number of items to display per row.
+   * @default 3
+   */
+  itemsPerRow?: 'wrap' | number;
+
+  /**
+   * Styles for the picker.
+   */
+  styles?: _VideoBackgroundEffectsPickerStyles;
+}
+
+/**
+ * Option for the {@link _VideoBackgroundEffectsPicker}.
+ * @internal
+ */
+export type _VideoBackgroundEffectChoiceOption = _VideoEffectsItemProps;
+
+/**
+ * Styles for the {@link _VideoBackgroundEffectsPicker}.
+ * @internal
+ */
+export interface _VideoBackgroundEffectsPickerStyles {
+  /**
+   * Styles for the root element.
+   */
+  root?: IStyle;
+
+  /**
+   * Styles for the label.
+   */
+  label?: IStyle;
+
+  /**
+   * Styles for the root of each row element.
+   */
+  rowRoot?: IStyle;
+}
+
+/**
+ * Picker for choosing a Video Background Effect.
+ *
+ * @remarks
+ * This functions similar to a radio group of buttons, where the user can select one of the options.
+ *
+ * @internal
+ */
+export const _VideoBackgroundEffectsPicker = (props: _VideoBackgroundEffectsPickerProps): JSX.Element => {
+  const [componentControlledSelectedEffectKey, setComponentControlledSelectedEffectKey] = React.useState<
+    string | undefined
+  >(props.selectedEffectKey);
+
+  const selectedEffect = props.selectedEffectKey ?? componentControlledSelectedEffectKey;
```

Comment:
If the selectedEffect provided is not present/removed should we console warn like when a pinned participant id is not one of the remote participants as shown [here](https://github.com/Azure/communication-ui-library/blob/5e4689854bd10f701b18a897607e1c5204454358/packages/react-components/src/components/VideoGallery.tsx#L337)?

Reply:
This is an excellent comment! Added. This also pointed me to this: https://github.com/microsoft/fluentui/blob/306fe5e1e38b36e84daf48a23817704416be2bb4/packages/utilities/src/warn/warnControlledUsage.ts#L31. Turns out fluent exports a component for warning against bad controlled component use! It has other warnings too in `useWarning` and only logs in development mode!